modules/lib/private-endpoint/main.bicep

```diff
@@ -0,0 +1,65 @@
+@description('Location for all Private Endpoint(s).')
+param location string
+
+@description('Array of Private Endpoint(s) to create.')
+param privateEndpoints array = []
```

Comment:
I think it make more sense to just create one pep in a module. It makes the code alot cleaner and easier to maintain.
Users can always use `for` loop on the module.

Reply:
Intention for this module is to cut down on duplicate code across all the other base modules, so moving that `for` logic into each module deviates away from that goal.

Reply:
I have refactored the module in this direction now.